airbyte-ci/connectors/connectors_insights/pyproject.toml

```
@@ -0,0 +1,37 @@
+[tool.poetry]
+name = "connectors-insights"
+version = "0.1.0"
+description = ""
+authors = ["Airbyte <contact@airbyte.io>"]
+readme = "README.md"
+packages = [{ include = "connectors_insights", from = "src" }]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+dagger-io = "0.9.6"
+click = "^8.1.7"
+pydantic = "^1.9"
+asyncer = "^0.0.7"
+asyncclick = "^8.1.7.2"
+connector-ops = { path = "../connector_ops", develop = false }
+beautifulsoup4 = "^4.12.3"
+requests = "^2.32.3"
+google-cloud-storage = "^2.17.0"
+
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry.scripts]
+connectors-insights = "connectors_insights.cli:connectors_insights"
+
+[tool.poetry.group.dev.dependencies]
+ruff = "^0.2.1"
+mypy = "^1.8.0"
+
+[tool.ruff.lint]
+select = ["I"]
+
+[tool.poe.tasks]
+type_check = "mypy src --disallow-untyped-defs"
+lint = "ruff check src --fix"
```

```suggestion
lint = "ruff check src --fix"

```
Look ma I'm helping.